db/seeders/dev_team_seeder.rb

```diff
@@ -40,9 +40,11 @@ def find_or_create_business_unit(attributes)
 
   #rubocop:disable Metrics/MethodLength
   def add_business_units
-    @foi = CorrespondenceType.foi
-    @sar = CorrespondenceType.sar
-    @ico = CorrespondenceType.ico
+    @foi      = CorrespondenceType.foi
+    @sar      = CorrespondenceType.sar
+    @ico      = CorrespondenceType.ico
+    @offender = CorrespondenceType.offender_sar
```

Comment:
Should be `@offender_sar` ?

Reply:
The abbreviation shouldn't be OFFENDER_SAR until your branch is merged :)